### DISCUSSION TITLE
Redirect gits.github.com to gist.github.com

### DISCUSSION THREAD

**Discussion by anewuser** [community/community#27519] [Other Feature Feedback, Questions, & Ideas]
`gits.github.com` is a common typo of `gist.github.com`. 

People who type `gits.github.com` into an address bar always, without any exception, want to see `gist.github.com` instead.

Years ago, a kind soul created the `gits.github.com` repository and made it redirect to its proper destination.

Now, this redirect is broken.

Please fix this by redirecting `gits.github.com` to `gist.github.com` again with no extra steps or unnecessary messages.

The owner of the old `gits.github.com` repository [is completely fine with this](https://github.com/community/community/discussions/27519#discussioncomment-3273518), since **this was his idea all along**. 

The old repository, which only existed for this redirection, can be ignored or deleted altogether. The user who created it will delete it by himself right away if you ask him. We just want the redirection to happen, and it will be handled solely and exclusively by Github from now on.

Hundreds or thousands of users making that typo weekly will thank you.

**Comment by ichenglin** [comment]
The message you quoted explains this quite well, the url you entered **goes through a third-party personal site**, which might lead to **security concerns**. Although at this moment it decides to redirect its visitors to the official site, there's **no guarantee** that in the future the site would still **remain as it was or no malicious content would be added**.

Despite the reason mentioned above, the typo url **were never an official link** anyway. I don't see a point in doing such decision.

**Reply by anewuser** [reply]
You've misinterpreted my post. I've rewritten it as clearly as possible. Please read it again.

**Comment by LumitoLuma** [comment]
@ichenglin is right. It is just the standard 404 error in GitHub, and they send you to [gits.github.io](https://gits.github.io), which shows the content of [this repo](https://github.com/gits/gits.github.com). Even though they could make that redirection automatic from [gits.github.com](https://gits.github.com) to [gist.github.com](https://gist.github.com), they won't do so.

**Reply by anewuser** [reply]
You've misinterpreted my post. I've rewritten it as clearly as possible. Please read it again.

**Comment by NV** [comment]
If somebody makes a typo in gist.github.com, it would be nice to redirect to the correct URL. "gits" was a common typo for me, so I made that account when GitHub pages were on github.com subdomains.

**Comment by jfullstackdev** [comment]
it's already 404

![image](https://user-images.githubusercontent.com/47092464/182760820-bda0f7cd-7caf-4dd1-999b-2ff82b7cbfff.png)

**Reply by anewuser** [reply]
That's exactly the point of this issue.

**Reply by jfullstackdev** [reply]
if a certain link is causing some unusual redirection for the benefit of another site, then it will be reported for sure

**Reply by anewuser** [reply]
@jdevstatic  please understand that gits.github.com is only ever accessed by people trying to get to https://gist.github.com/ . It's a typo.

This is what I'm suggesting here: make it a straightforward redirection. The person who used to control gits.github.com years ago [is totally fine with my suggestion, since this was his original idea too](https://github.com/orgs/community/discussions/27519#discussioncomment-3273518).

**Reply by jfullstackdev** [reply]
but there exist that user and the repo, so I think, that's not allowed, 

![image](https://user-images.githubusercontent.com/47092464/182765153-3e972c5d-caa7-4fe5-a0e5-721a3aa7dca1.png)

and there is the confusion on the description, it's intentional not typo, anyways, that was 13 years ago

**Reply by anewuser** [reply]
@jdevstatic  that user (NV) left a comment on this thread (this one, the one that you're reading right now). I even linked to it. He agrees with my idea. He wants my idea to be implemented. That's why he created the repository in the first place years ago. Back then, he made it redirect to the proper destination. Now it doesn't work anymore, so Github can delete the old repository, and redirect the URL directly to the proper destination for the benefit of all. This is all we're asking.
